keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
```

Comment:
Are we sure that we want to handle all these numpy objects and numpy TF experimental wrapped TF ops instead of handling directly Tensor and TF obs?

Reply:
I am asking this also for coherence:

```bash
grep -r "import numpy" * --exclude="*test.py" --exclude-dir="*examples*" --exclude-dir="*test*" --exclude-dir="*benchmarks*"
```
```bash
keras_cv/layers/preprocessing/random_rotation.py:import numpy as np
keras_cv/models/object_detection/__internal__.py:import numpy as np
keras_cv/models/object_detection/retina_net/retina_net.py:import numpy as np
```

Then also in these few cases it was used only for `np.log` and `np.pi`. 
(I don't know if there was a really rationale cause we could not use pi from python `math`/` tf.math.log`)